src/services/qrProcessor/index.js

```diff
@@ -0,0 +1,19 @@
+import { get } from "axios";
+
+export const decodeQrCode = qrCode => {
+  const ttRegex = /tt:\/\/(.*)/;
```

Comment:
tt:// is too generic there's probably some other app in the wild using it
go with tradetrust://
also namespace the action because if there's a mobile app in the future we'll probably use the deeplinking feature for more things than this
so
tradetrust://view-document?target=\<encoded url\>

refer to
https://portal.deeplink.me/documentation 
under schemes and url handling

Reply:
Changed to tradetrust